lib/csafe/command.js

```diff
@@ -33,7 +33,7 @@ class Command extends Frame {
     // data byte count and data.
     if (command.type == 'long') {
       bufferArray.push(data.length)
-      bufferArray.concat(data)
+      bufferArray = bufferArray.concat(data)
```

Comment:
[`Array.prototype.concat()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) returns a new array rather than mutating the instance.